src/preset_cli/auth/main.py

```diff
@@ -46,6 +48,8 @@ def reauth(self, r: Response, *args: Any, **kwargs: Any) -> Response:
         if r.status_code != 401:
             return r
 
+        _logger.info("Token expired. Re-authenticating...")
```

Comment:
Could this be a `DEBUG` instead? I think it's useful when debugging but we might not need it logged for all executions.

Reply:
Sure, changed to debug.

Reply:
Thanks!